index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<title>Portfolio</title>
+</head>
+<body>
+	<section class="screen_container">
+		<header>
+			<nav>
+				<section class="toolbar_container">
+					<p class="Title_logo">My Logo</p>
+					<img src="./images/i.png"  class="logo" alt="Icon menu">
```

Comment:
- [ ] Nice work wrapping the hamburger button image with `nav` tag :clap: However what do you think about using this structure for the Toolbar:
```sh
<header class="toolbar_container">
  <a href="#" class="Title_logo">My Logo</a>
  <nav>
    <img src="./images/i.png"  class="logo" alt="Icon menu">
  </nav>
</header>
```
It is now cleaner, isn't it? :sunglasses: